File: AutomatonDfa/src/States.java
import java.security.PublicKey;
import java.util.ArrayList;

/**
 * Created by jenamorado on 7/25/16.
 */
public class States {
    public String name;
    public ArrayList<Transition> transitions;
    boolean isInitialState;
    boolean isAcceptanceState;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public boolean isInitialState() {
        return isInitialState;
    }

    public void setInitialState(boolean initialState) {
        isInitialState = initialState;
    }

    public boolean isAcceptanceState() {
        return isAcceptanceState;
    }

    public void setAcceptanceState(boolean acceptanceState) {
        isAcceptanceState = acceptanceState;
    }


}

File: AutomatonDfa/src/States.kt
import java.security.PublicKey
import java.util.ArrayList

/**
 * Created by jenamorado on 7/25/16.
 */
class States {
    var name: String? = null
    var transitions: ArrayList<Transition>? = null
    var isInitialState: Boolean = false
    var isAcceptanceState: Boolean = false
}
